docs/sbpy/data.rst

```diff
@@ -5,17 +5,328 @@ Introduction
+    ...                          [2451524.14653, 2451524.23541]*u.d,
+    ...                          ['r', 'z']],
+    ...                         names=['ra', 'dec', 't', 'filter'])
+    >>> obs.add_rows(obs2)
+    7
+
+Individual elements, entire rows, and columns can be modified by
+directly addressing them:
+
+    >>> print(obs['ra'])
+    [10.223423 10.233453 10.243452 10.25546  10.265425 10.4545   10.5656  ] deg
+    >>> obs['ra'][:] = obs['ra'] + 0.1*u.deg
+    >>> print(obs['ra'])
+    [10.323423 10.333453 10.343452 10.35546  10.365425 10.5545   10.6656  ] deg
+
+Note the specific syntax in this case (``obs['ra'][:] = ...``) that
+is required by `~astropy.table.Table` if you want to replace
+an entire column.
+    
+More complex data table modifications are possible by directly
+accessing the underlying `~astropy.table.QTable` object.
+
+Writing object data to a file
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`~sbpy.data.DataClass` objects can be written to files using
+`~sbpy.data.DataClass.to_file`:
+
+    >>> obs.to_file('observations.dat')
+
+By default, the data are written in ASCII format, but other formats
+are available, too (cf. `~astropy.table.Table.write`).
+
+  
+How to use Orbit
+----------------
+tbd
+
+
+How to use Ephem
+----------------
+tbd
+
+
+How to use Phys
+---------------
+tbd
+
+
+How to use Names
+----------------
+
+`~sbpy.data.Names` is different from the other classes in `~sbpy.data`
+in that it does not use `~sbpy.data.DataClass` as a base class. Instead,
+`~sbpy.data.Names` does not contain any data, it merely serves as an
+umbrella for functions to identify asteroid and comet names, numbers,
+and designations.
+
+In order to distinguish if a string designates a comet or an asteroid,
+you can use the following code:
+
+    >>> from sbpy.data import Names
+    >>> print(Names.asteroid_or_comet('(1) Ceres'))
+    asteroid
+    >>> print(Names.asteroid_or_comet('2P/Encke'))
+    comet
+
+The module basically uses regular expressions to match the input
+strings and find patterns that agree with asteroid and comet names,
+numbers, and designations. There are separate tasks to identify
+asteroid and comet identifiers:
+
+    >>> print(Names.parse_asteroid('(228195) 6675 P-L'))
+    {'number': 228195, 'desig': '6675 P-L'}
+    >>> print(Names.parse_asteroid('C/2001 A2-A (LINEAR)')) # doctest: _ELLIPSIS
+    ...sbpy.data.names.TargetNameParseError: C/2001 A2-A (LINEAR) does not appear to be an asteroid identifier
+    >>> print(Names.parse_comet('12893')) # doctest: +ELLIPSIS
+    ...sbpy.data.names.TargetNameParseError: 12893 does not appear to be a comet name
+    >>> print(Names.parse_comet('73P-C/Schwassmann Wachmann 3 C	'))
+    {'type': 'P', 'number': 73, 'fragment': 'C', 'name': 'Schwassmann Wachmann 3 C'}
+    
+Note that these examples are somewhat idealized. Consider the
+following query:
+
+    >>> print(Names.parse_comet('12893 Mommert (1998 QS55)'))
```

Comment:
I think the main issue here is that this string is nonstandard.  Isn't the IAU convention to write the number in parentheses and drop the temporary designation after it is named?  I accept that this is not an unusual way to write the asteroid name, but is simply just not the convention.